site/sfguides/src/terraforming_snowflake/terraforming_snowflake.md

```diff
@@ -374,8 +363,8 @@ You're almost done with the demo. We have one thing left to do: clean up your ac
    ```
     $ terraform destroy
    ```
-1.  Accept the changes if they look appropriate.
-1. Log in to the console to verify that all the objects are destroyed. The database, schema, warehouse, role, and the user objects created by Terraform will be automatically deleted.
+2.  Accept the changes if they look appropriate.
```

Comment:
It's up to you, but I think it's easier to keep numeric lists as 1. 1. 1. 1. 1. because you can add items later and not renumber the sequence...  When rendered they automatically go to 1. 2....

Reply:
i didn't know that. i changed it back